utils/frame/generate-bags/node-runtime/src/main.rs

```diff
@@ -17,30 +17,31 @@
 
 //! Make the set of bag thresholds to be used with pallet-bags-list.
 
+use clap::Parser;
 use generate_bags::generate_thresholds;
 use std::path::PathBuf;
-use structopt::StructOpt;
 
-#[derive(Debug, StructOpt)]
+#[derive(Debug, Parser)]
+#[clap(author, version, about)]
```

Comment:
```suggestion
```

Do we need this?

Reply:
without `#[clap(author, version, about)]`: 
```
node-runtime-generate-bags 

USAGE:
    node-runtime-generate-bags [OPTIONS] --total-issuance <TOTAL_ISSUANCE> --minimum-balance <MINIMUM_BALANCE>>
...
```

with `#[clap(author, version, about)]`: 

```
node-runtime-generate-bags 3.0.0
Parity Technologies <admin@parity.io>
Bag threshold generation script for pallet-bag-list and node-runtime.

USAGE:
    node-runtime-generate-bags [OPTIONS] --total-issuance <TOTAL_ISSUANCE> --minimum-balance <MINIMUM_BALANCE>>
...
```

I don't think there's any harm in adding this line.